<launch>

<arg name="enable_bag" default="false"/>

<include file="$(find infi_learn)/launch/episode_terminator.launch">
    <arg name="reset_service" value="velocity_filter/reset"/>
    <arg name="reward_topic" value="ape/reward"/>
    <arg name="min_reward" value="4.5"/>
</include>

<include file="$(find infi_feat)/launch/vo_features.launch">
    <arg name="image_topic" value="camera_ids/image_raw"/>
    <arg name="odom_topic" value="vel_odom"/>
</include>

<include file="$(find infi_perf)/launch/rewards_vel2d.launch">
    <arg name="odom_topic" value="odom_throttled"/>
</include>

<node pkg="rosbag" type="record" name="bagger" output="screen" if="$(arg enable_bag)"
args="-O /tmp/vo_features.bag 
    feature_image
    belief_state/stream_raw
    ape/reward
    vo_interface/normalized_parameters
    terminator/breaks"/>

</launch>